test/controllers/JourneyAnswersControllerSpec.scala

```diff
@@ -185,6 +186,27 @@ class JourneyAnswersControllerSpec extends ControllerBehaviours with ScalaCheckP
     }
   }
 
+  "getIncomePrepopAnswers" should {
```

Comment:
There are only success cases being tested here, is it because error handling has a TODO on it? Maybe we should look to do the proper error handling at some point soon. All Left's are being mapped to a 500s atm.

Reply:
Spoke with Lukasz, these will all likely be replaced with a generic service we shall test more comprehensively once we have them all sorted and start on error handling. We'll then add all the tests for error cases